week-2-wide-perspective/w02e02-data-manipulation/src/orders/tiles/AnnualOrdersTile.js

```diff
@@ -1,25 +1,36 @@
-const componentId = 'annual-orders'
-const mountPoint = document.querySelector(`[data-tile="${componentId}"]`)
-const subTitle = mountPoint.querySelector('[data-subtitle]')
-const panel = mountPoint.querySelector('[data-panel]')
+import { axiosClient } from '../../utils/client';
+import { $click } from './AddOrderButton';
+
+const componentId = 'annual-orders';
+const mountPoint = document.querySelector(`[data-tile="${componentId}"]`);
+const subTitle = mountPoint.querySelector('[data-subtitle]');
+const panel = mountPoint.querySelector('[data-panel]');
 
 // Uważaj na odfiltrowanie wg. roku (2022)
-subTitle.textContent = 'Year 2022'
-panel.innerHTML = ''
+subTitle.textContent = 'Year 2022';
+panel.innerHTML = '';
 
-// Tutaj podobnie, powinniśmy interpretować dane z: ordersFakeData
-for (const orderDate of [
-  '2022-10-06T15:03:25.132Z',
-  '2022-10-16T17:43:26.300Z',
-  '2022-11-07T02:37:43.525Z',
-]) {
-  panel.appendChild(makeLiElement({ orderDate, orderNumber: '02/2022' }))
+const updateTile = async () => {
+  panel.innerHTML = ''
```

Comment:
ten wpis:
```javascript
panel.innerHTML = ''
```

powinien być poniżej - jak już zaczynamy manipulację `DOM` - tak to się w `GUI` robi taki nieprzyjemny "skok" w `UI` i widać że znikają nam kontrolki co kliknięcie w button.